### PR TITLE
Fix text structure for DATA command

### PR DESCRIPTION
**Pull request by matristle**
-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Callbacks command: Update verbiage`

-   [ ] I have ensured all tests related to any command files included in this PR pass, and/or all snapshots are up to date

<hr>

**1. Because:**
  - The text structure wasn't consistent (new line, missing quotation mark, and ``` wasn't used for the error message)
  
**2. This PR:**
Makes the text structure more consistent.

**3. Additional Information:**
I noticed that there was a comment in there saying:

```js
  // this weird formating is needed because of some indentation on mobile
```
but I didn't want to assume for which it was meant, so I roughly tested it privately on mobile, but there was no indentation issue there. Nevertheless, there might be that issue when the content is in a card on the actual outcome.

### CODE REVIEW

File: botCommands/simpleCommands.js [hunk 1]
```diff
@@ -52,16 +52,17 @@ const data = {
       .setDescription(`
 Instead of asking if anyone can help you, ask your question outright so people can help you!
 
-**Bad**: "Hey, does anyone know how to set CSS styles with Javascript?"
+**Bad**:
+"Hey, does anyone know how to set CSS styles with Javascript?"
 
 **Good**:
-"Hey, I'm having trouble setting CSS styles via Javascript.
+"Hey, I'm having trouble setting CSS styles via Javascript."
```
Comment (marvingay) [inline botCommands/simpleCommands.js:59]: I don't believe this closing double quote goes here as it is meant to encapsulate the entire message for the "Good" example.

**Review by marvingay — CHANGES_REQUESTED** [review]

File: botCommands/simpleCommands.js (hunk 1 above)
Comment (matristle) [reply botCommands/simpleCommands.js:59]: Of course, I see that now.

**Review by marvingay — CHANGES_REQUESTED** [review]
The snapshot needs to be updated such that the tests pass.

**Comment by zachmmeyer** [comment]
@matristle you can fix the snapshot in this PR still

**Pull request closed without merging**